content/v2.0/reference/api/client-libraries/python.md

```diff
@@ -27,9 +27,11 @@ If just getting started, see [Getting started with InfluxDB](/v2.0/get-started/)
     ```
 
 2. Ensure that InfluxDB is running.
-   If running InfluxDB locally, visit http://localhost:9999.
-   (If using InfluxDB Cloud, visit the URL of your InfluxDB Cloud UI.
-   For example: https://us-west-2-1.aws.cloud2.influxdata.com.)
+If using InfluxDB Cloud, visit the URL of your InfluxDB Cloud UI.
+For example: https://us-west-2-1.aws.cloud2.influxdata.com.
+_For specific InfluxDB Cloud provider and region URLs, see [InfluxDB Cloud URLs](/v2.0/cloud/urls/)._
+(If running InfluxDB locally, visit http://localhost:9999.)
```

Comment:
I have a thought about something to do with the Cloud URLs doc. I'll ping you Slack.